pom.xml

```diff
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudbees</groupId>
     <artifactId>cloudbees-oss-parent</artifactId>
-    <version>5</version>
+    <version>6</version>
```

Comment:
You can ignore this, should be the same values but I had to merge master to my branch.